### DISCUSSION TITLE
FreedomWave 0.9.7 Beta

### DISCUSSION THREAD

**Discussion by RChadwick7** [RChadwick7/FreedomWavePD#5] [General]
This release finally implements the menu system, with a lot of variables to change. Be aware, the function to 'Load Defaults' doesn't work, as well as the 'Intensity'. All other settings should work and adjust in real time, even during treatment. I still need to document all the new features. Until that's done, just ask.

<hr /><em>This discussion was created from the release <a href='https://github.com/RChadwick7/FreedomWavePD/releases/tag/'>FreedomWave 0.9.7 Beta</a>.</em>

**Comment by ecomian** [comment]
How do I install  the  update on my Arduino? clicking avrdude.exe does no thing and  running "Program-the-FreedomWave.bat " produces  error - 
"avrdude stk500_recv() error: programmer is not responding
avrdude stk500_getsync() warning: attempt 1 of 10: not in sync: resp=0x00
avrdude stk500_recv() error: programmer is not responding"

**Reply by RChadwick7** [reply]
Depending on the bootloader in your Arduino, an error message is normal. The script first tries communicating with the Arduino at a specific baud rate. If it doesn't work, it tries the second baud rate. Do you see a bar graph or a message that it completed successfully?

**Reply by ecomian** [reply]
Thanks for the help. As it turn out, I finally got it going. In my training as an electronics technician I learned some Basic programing. 
That was 40yrs ago and have not used it since.  According to your REM comment, I added to port name manually replacing chgport instructs, thus:

Deleted: 

chgport > port.txt
set /p PORTLINE= < port.txt
set PORTNAME=%PORTLINE:~0,5%
echo Serial port found at %PORTNAME%

Replacing with:
set PORTNAME=COM4 

Deleting the 

I edited the .bat file from 

That worked.  I don't know if that is the right way to do it.
